lib/roby/app/rake.rb

```diff
@@ -342,7 +378,52 @@ def use_junit?
                 # Path to the JUnit/Rubocop reports (if enabled)
                 attr_accessor :report_dir
 
-                def run_roby_test(*args, report_name: "report", coverage_name: "roby")
+                def spawn_process_capturing_output(bin, *args)
+                    stdout_r, stdout_w = IO.pipe
+                    pid = spawn(Gem.ruby, bin, *args, out: stdout_w, err: stdout_w)
+                    stdout_w.close
+                    [pid, stdout_r]
+                end
+
+                def read_captured_output_from_pipe(pid, read_pipe)
+                    output = []
+                    begin
+                        while (output_fragment = read_pipe.read(512))
+                            output << output_fragment
+                        end
+                        output.join ""
+                    rescue Interrupt
+                        Process.kill "TERM", pid
```

Comment:
Why TERM ? Shouldn't it be ... INT ?

Reply:
I didnt realize it, just kept the same code as before, I will update to use INT